nlp/2021_04_22_shap_for_huggingface_transformers/README.md

```diff
@@ -0,0 +1,25 @@
+# BERT, explain yourself! 📖
+
+It has been over two years since transformer models took the NLP throne 🏅, but up until recently they couldn't tell you how they got to their prediction. Not anymore!
+
+```python
+>>> BERT('After many years of training I finally succeeded in achieving '
```

Comment:
Might be nice to use an example sentence about the post itself ? Smtg like "After many year of text classification SHAP now allows us to dig in the model's logic! This great tool... " 😃

Reply:
I think this is a nice suggestion, but given all the changes required I will leave it in the current example.